portfolio/src/main/webapp/style.css

```diff
@@ -15,6 +15,12 @@ h2 {
   font-family: Helvetica;
 }
 
+h3 {
+  font-size: 25px;
+  color: rgba(128, 128, 128, 0.24);
+  font-family: Helvetica;
+}
```

Comment:
You probably don't have to worry about this for now, given the small and self-contained nature of the project--but in practice you should almost always prefer [class selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors) to [type selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Type_selectors).

The reasons are somewhat involved, but if you're interested and/or expect to write a lot of CSS, I would be happy to elaborate further.

Reply:
I'm not sure how much CSS I'll be writing, but I am always interested in learning new stuff, if you have time to explain!

Reply:
The basic gist is that CSS rules are effectively global. Once an HTML document loads a stylesheet, every rule declared in that stylesheet is applied to the entire document. There is no way to narrow the scope of the stylesheet to some sub-tree of the DOM.

The problem with type selectors is that they are overly general. An `h3` rule will apply to every `<h3>` on the page. But generally, an element's style depends on the surrounding context. An `<h3>` in the main content of an article will look different from an `<h3>` in a side nav, pop-up dialog, or block quote. Class selectors offer a way to achieve this by selectively targeting specific elements.

In a small single-page app, primarily using type selectors will probably serve you just fine (if it ain't broke, don't fix it). Just know that it's a best practice to use class selectors, especially in larger projects.

Reply:
Ah, ok. That makes sense! Thank you for explaining that to me, I appreciate it!